WooCommerce/StoreWidgetsIntentsExtension/IntentHandler.swift

```diff
@@ -0,0 +1,34 @@
+import Intents
+
+class IntentHandler: INExtension, StoreWidgetsConfigIntentHandling {
```

Comment:
Maybe some docs, explaining how this works would be useful!

Reply:
Added in 5d0d531.